flake/zombienet.nix

```diff
@@ -97,11 +117,26 @@
           program = self'.packages.zombienet-dali-complete;
         };
 
+        zombienet-dali-centauri-a = {
+          type = "app";
+          program = self'.packages.zombienet-dali-centauri-a;
+        };
+
         zombienet-picasso-complete = {
           type = "app";
           program = self'.packages.zombienet-picasso-complete;
         };
 
+        zombienet-log-follow = {
+          program = pkgs.writeShellApplication rec {
+            name = "zombienet-log-follow";
+            text = ''
+              docker exec -it composable-devnet-a-1   bash  -c 'LOG=$(find /tmp/ -name "zombie-*" | head --lines=1)/alice.log && tail --follow $LOG'
```

Comment:
nice

Reply:
why only Alice?

Reply:
only alice.log node has well know port 9988, other nodes has random ports and will not be published outside

Reply:
so likely nothing will try to connect to bob.log